editoast/src/views/rolling_stocks.rs

```diff
@@ -0,0 +1,68 @@
+use crate::error::Result;
+use crate::schema::rolling_stock::RollingStock;
+use crate::DbPool;
+use actix_web::dev::HttpServiceFactory;
+use actix_web::get;
+use actix_web::web::{self, block, Data, Json, Path};
+
+pub fn routes() -> impl HttpServiceFactory {
+    web::scope("/rolling_stock").service(get)
+}
+
+/// =================================
+/// Endpoints
+/// =================================
```

Comment:
This is not a good idea. I think you take this from `electrical_profiles.rs`. It's not a good idea. It's actually doing something weird on the documentation.